test/cross-chain/cross-chain.js

```diff
@@ -154,46 +152,42 @@ contract("Cross-chain", (accounts) => {
     // in to a different location. If we see another chain id, we assume it's non-coverage
     // truffle and look for the build artifacts in the normal place.
 
-    let homeEtherRouterAddress;
-    let foreignEtherRouterAddress;
-    if (process.env.HARDHAT_FOREIGN === "true") {
-      homeEtherRouterAddress = newEtherRouterAddress;
-      foreignEtherRouterAddress = (await EtherRouter.deployed()).address;
-    } else {
-      homeEtherRouterAddress = (await EtherRouter.deployed()).address;
-      foreignEtherRouterAddress = newEtherRouterAddress;
-    }
-
-    console.log("foreign colony network", foreignEtherRouterAddress);
-    console.log("home colony network", homeEtherRouterAddress);
-
+    const homeEtherRouterAddress = (await EtherRouter.deployed()).address;
     homeColonyNetwork = await new ethers.Contract(homeEtherRouterAddress, IColonyNetwork.abi, ethersHomeSigner);
+
+    const foreignEtherRouterAddress = homeEtherRouterAddress;
     foreignColonyNetwork = await new ethers.Contract(foreignEtherRouterAddress, IColonyNetwork.abi, ethersForeignSigner);
   });
 
   beforeEach(async () => {
+    console.log("beforeEach");
```

Comment:
I assume these should be deleted

Reply:
I could have sworn I got rid of these, so I probably did on another version of this branch I had checked out somewhere and never pushed...